webroot/js/self-service.js

```diff
@@ -13,7 +13,7 @@
  */
 foodcoopshop.SelfService = {
 
-    autoLogoutTimer : 180,
+    autoLogoutTimer : 5,
```

Comment:
den wert hier bitte nicht einfach überschreiben, das betrifft ja sonst alle. bitte über eine neue config in app_config.php (kann dann in custom_config.php überschrieben werden) lösen
`app.selfServiceModeAutoLogoutTimer = 180`;